### PR TITLE
Windows Fixes

### PR DESCRIPTION
**Pull request by marsupial**

### CODE REVIEW

File: lib/Interpreter/LookupHelper.cpp [hunk 1]
```diff
@@ -387,9 +387,9 @@ namespace cling {
                                 quickTypeName.size()-6);
       innerConst = true;
     }
-    constexpr int pointerType = 0;
-    constexpr int lrefType = 1;
-    constexpr int rrefType = 2;
+
+    enum PointerType { kPointerType, kLRefType, kRRefType, };
```
Comment (pcanal) [inline lib/Interpreter/LookupHelper.cpp]: How is this change related to noexcept/constexpr?  If it isn't, can we split it into its own commit?  Thanks.

Comment (marsupial) [inline lib/Interpreter/LookupHelper.cpp]: Only related in prior history of changes that introduced the use of constexpr.
Splitting it out should be easy enough.
